lib/bulbs-elements/util/in-view-monitor.js

```diff
@@ -96,6 +100,11 @@ const maybeEmitEvent = (monitoredItem) => {
     let event = new CustomEvent('exitviewport');
     monitoredItem.element.dispatchEvent(event);
   }
+
+  if(!inView && approachingViewport) {
+    let event = new CustomEvent('approachingviewport');
```

Comment:
We haven't namespaced any of our other events from InViewMonitor, but that could make sense. Not sure if we want to take on that general improvement here though.

Any ideas what a good namespace would be?